Automattic/wp-calypso
# Help Center: Show support chat if user is admin and non owner on atomic site

Related to https://github.com/Automattic/wp-calypso/issues/89984

## Proposed Changes

* These changes check if the user is on an atomic site and the user is an administrator on the site to grant them support within the Help Center.

## Why are these changes being made?


* Administrators on Atomic sites who are not site owners are currently not able to access the Zendesk chat. Instead, they are redirected to the forum form.

## Testing Instructions



* Checkout branch
* Sandbox public api and test site.
* Navigate to `apps/editing-toolkit`, build the code, and compress the `editing-toolkit-plugin` folder.
* Sync the code to your sandbox.
* Add a new user to the site with an administrator role.
* Open the Help Center and verify the `Still need help` button navigates to the correct support screen.

## Pre-merge Checklist
